resources/faq.md

```diff
@@ -0,0 +1,163 @@
+# FAQ
+
+This FAQ is still a work in progress.<br>
+Do not hesitate to ask us your own questions. It will help us improve this page!
```

Comment:
If you have any questions we want to hear from you. Your feedback will help us improve this page!